daemon/network/settings.go

```diff
@@ -2,11 +2,13 @@ package network
 
 import "github.com/docker/docker/pkg/nat"
 
+// Address represents an IP address
```

Comment:
This isn't really sufficient. It should provide information about why this type exists, why we've opted away from using standard types (ie `net.IP`). It is also missing documentation for the fields.